bin/damsolrizer

```diff
@@ -101,12 +101,25 @@ begin
       max_tries = 2
       puts @msg.headers.inspect
       puts "PID: #{@msg.headers["pid"]}"
+      puts "method: #{method}"
       begin
-        if ["addDatastream", "addRelationship","ingest","modifyDatastreamByValue","modifyDatastreamByReference","modifyObject","purgeDatastream","purgeRelationship"].include? method
+        if ["modifyCollection", "addDatastream", "addRelationship","ingest","modifyDatastreamByValue","modifyDatastreamByReference","modifyObject","purgeDatastream","purgeRelationship"].include? method
           solrizer = Solrizer::Fedora::Solrizer.new
           solrizer.solrize @msg.headers["pid"], solropts
           puts  "updated solr index for #{@msg.headers["pid"]}\n"
-        elsif method == "purgeObject"
+          if method == "modifyCollection"
+            msg = Dams::ShareApi.share_push(pid)
+            if msg == "success"
+              puts  "Pushed #{pid} to OSF\n"
+            elsif msg == "failed"
+              puts  "Document is not valid, #{pid} is failed to push to OSF\n"
```

Comment:
I think this is great. At the moment since we're targeting collections, should we use `Collection` instead of `Document`? I'm just wondering if that will be more clear to end users.